scipy/stats/_continuous_distns.py

```diff
@@ -883,6 +883,12 @@ class burr_gen(rv_continuous):
     -----
     The probability density function for `burr` is:
 
+    .. math::
+
+        f(x, c, d) = c d x^{c - 1} / (1 + x^{c})^{d + 1}
+        
+    Also equivalent to:
+
     .. math::
 
         f(x, c, d) = c d x^{-c - 1} / (1 + x^{-c})^{d + 1}
```

Comment:
These are only equivalent for `d=1`, i.e. for the fisk distribution.

Reply:
This is the pdf formula for burr distribution, so the first one is the right one ?

![image](https://user-images.githubusercontent.com/54520113/117561357-7cc80b80-b096-11eb-90be-dbda28087ed5.png)

Reply:
I agree, looks like the first one is the right one. Also, if we're using LaTeX anyway, let's use `\frac` and make it look identical to how the pdf is given on Wikipedia.

Reply:
![image](https://user-images.githubusercontent.com/6570539/155265857-1b4c034a-77c0-47f8-9aa2-f36d0d03c4a5.png)

Reply:
The second one, the original one, is the intended one for the `burr` distribution. 

> This is the PDF corresponding to the third CDF given in Burr's list;
    specifically, it is equation (11) in Burr's paper [1]_

This is:
![image](https://user-images.githubusercontent.com/6570539/155267001-89e8ca81-423e-4778-8c5e-c258ad2e3edf.png)

Taking the derivative and changing the symbols used for the parameters, it turns into:
![image](https://user-images.githubusercontent.com/6570539/155267172-cced3377-06fc-4ee2-807a-f82464c39054.png)